packages/fault-detector/src/service.ts

```diff
@@ -124,7 +124,7 @@ export class FaultDetector extends BaseServiceV2<Options, Metrics, State> {
       this.state.oo = {
         contract: oo,
         filter: oo.filters.OutputProposed(),
-        getTotalElements: async () => oo.latestOutputIndex(),
+        getTotalElements: async () => oo.nextOutputIndex(),
```

Comment:
Note the off-by-one that would cause an underflow and an error on system startup: https://github.com/ethereum-optimism/optimism/blob/576a8fcbfe0239df5691392af0613cd43ba09608/packages/contracts-bedrock/contracts/L1/L2OutputOracle.sol#L305-L307